src/ode23s.jl

```diff
@@ -56,15 +56,15 @@ function show(io::IO, state :: RosenbrockState)
 end
 
 
-function start{O<:ExplicitODE,S<:ModifiedRosenbrockStepper}(s :: Solver{O,S})
+function start{O<:ExplicitODE,S<:ModifiedRosenbrockStepper}(s::Solver{O,S})
     t  = s.ode.t0
     dt = s.options.initstep
     y  = s.ode.y0
     dy = zero(y)
 
     J  = Array(eltype(y),length(y),length(y))
 
-    step  = Step(t,deepcopy(y),deepcopy(dy))
+    step  = Step(t,copy(y),copy(dy))
```

Comment:
I would rather leave it as is and change it later, maybe once 0.5 is out  What do you think about the new version of Tableau?